### PR TITLE
Fix Html interpretation in help

### LINKED ISSUE #1950: [TRANSLATE] Help in infoboxes are only available in FR and DE!
Could also fix #1838 and #1896 in the same time

### PR DESCRIPTION
**Pull request by ponceta**
Fix #1950, #1838 and #1896

Solve html binding for help
(inclusive sce validation)
Allow it/en translations
Fix wms and kml help popups

Test [here](http://mf-geoadmin3.dev.bgdi.ch/ltpoa/)

### CODE REVIEW

**Comment by ponceta** [comment]
Please review :)

File: src/components/help/HelpDirective.js [hunk 1]
```diff
@@ -24,15 +24,17 @@
...
                                 'ng-repeat="res in options.results">' +
-                             '<h2 ng-bind="res[1]"></h2>' +
-                             '<div ng-bind="res[2]"></div>' +
+                             '<h2 ng-bind-html=trustAsHtml(res[1])></h2>' +
+                             '<div ng-bind-html=trustAsHtml(res[2])></div>' +
                              '<img ng-src="{{res[4]}}" ' +
                                   'draggable="false"/>' +
                            '</div>';
 
+        $rootScope.trustAsHtml = $sce.trustAsHtml;
```
Comment (oterral) [inline src/components/help/HelpDirective.js]: Usually we avoid attaching function to rootScope when it's not necessary, may be you can apply the trustAsHtml function when you receive results [here](https://github.com/geoadmin/mf-geoadmin3/blob/poa_html_interpretation_fix/src/components/help/HelpDirective.js#L106) 

```
results.push([
  $sce.trustAsHtml(res.rows[0][1],
  $sce.trustAsHtml(res.rows[0][2])
]);
                      
```

**Comment by oterral** [comment]
it seems the new line character is ignored by the `$sce` service, could you try to replace  `\n` by `<br/>`

File: src/components/help/HelpDirective.js (hunk 1 above)
Comment (ponceta) [inline src/components/help/HelpDirective.js]: Nice :) I'll try that :)

**Comment by ponceta** [comment]
Yes of course!

**Comment by oterral** [comment]
Perfect @ponceta!! thx for the fix!!

**Comment by ponceta** [comment]
I've still some cosmetics "by hand" to do on the fusion table but ty for the help, it was precious to me :)

**Pull request merged**